sdk/nodejs/iam/managedPolicies.ts

```diff
@@ -153,7 +153,7 @@ export module ManagedPolicies {
     export const AWSLambdaENIManagementAccess: ARN = "arn:aws:iam::aws:policy/service-role/AWSLambdaENIManagementAccess";
     /** Use ManagedPolicy.AWSLambdaExecute instead. */
     export const AWSLambdaExecute: ARN = "arn:aws:iam::aws:policy/AWSLambdaExecute";
-    /** Use ManagedPolicy.AWSLambdaFullAccess instead. */
+    /** @deprecated No longer supported. New IAM Roles created using this policy will fail. Use ManagedPolicy.AWSLambda_FullAccess instead. */
```

Comment:
```suggestion
    /** @deprecated No longer supported. New IAM Roles created using this policy will fail. Use ManagedPolicy.LambdaFullAccess instead. */
```

Reply:
I also noticed we don't do the same thing for `AWSLambdaReadOnlyAccess` which should also be updated a few lines down in this file.